examples/stable_diffusion_v2/ldm/modules/train/callback.py

```diff
@@ -156,7 +180,7 @@ def on_train_step_end(self, run_context):
                 self.step_start_time = time.time()
                 if self.record_lr:
                     _logger.info(
-                        "epoch: %d step: %d, lr: %.7f, loss: %.3f, loss scale: %d, average step time (in %d step(s)): %.3f.",
+                        "epoch: %d step: %d, lr: %.7f, loss: %.6f, loss scale: %d, average step time: %.6f.",
```

Comment:
since `(in %d step(s))` is deleted, then `self.log_interval` need also be deleted, otherwise it will raise error